<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QgsNewConnectionBase</class>
 <widget class="QDialog" name="QgsNewConnectionBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Create a New PostGIS connection</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnOk" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHelp" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>Help</string>
       </property>
       <property name="shortcut" >
        <string>F1</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="Q3GroupBox" name="GroupBox1" >
     <property name="title" >
      <string>Connection Information</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="TextLabel1_2" >
            <property name="text" >
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel1" >
            <property name="text" >
             <string>Host</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel2" >
            <property name="text" >
             <string>Database</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel2_2" >
            <property name="text" >
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel3" >
            <property name="text" >
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel3_2" >
            <property name="text" >
             <string>Password</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLineEdit" name="txtName" >
            <property name="toolTip" >
             <string>Name of the new connection</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtHost" />
          </item>
          <item>
           <widget class="QLineEdit" name="txtDatabase" />
          </item>
          <item>
           <widget class="QLineEdit" name="txtPort" >
            <property name="text" >
             <string>5432</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtUsername" />
          </item>
          <item>
           <widget class="QLineEdit" name="txtPassword" >
            <property name="echoMode" >
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkStorePassword" >
          <property name="text" >
           <string>Save Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnConnect" >
          <property name="text" >
           <string>Test Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends></extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtHost</tabstop>
  <tabstop>txtDatabase</tabstop>
  <tabstop>txtUsername</tabstop>
  <tabstop>txtPassword</tabstop>
  <tabstop>chkStorePassword</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnHelp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
